superset/assets/javascripts/explore/components/ChartContainer.jsx

```diff
@@ -75,6 +75,7 @@ class ChartContainer extends React.PureComponent {
     return {
       viewSqlQuery: this.props.queryResponse.query,
       containerId: props.containerId,
+      datasource: props.datasource,
```

Comment:
curious why we are using `this.props` and `props` in this method. would be nice to clean this up and use either `props` or `this.props` for consistency.